src/zenml/orchestrators/base_orchestrator.py

```diff
@@ -405,6 +423,70 @@ def _cleanup_run(self) -> None:
         self._active_deployment = None
         self._active_pb2_pipeline = None
 
+    def get_run_id_for_orchestrator_run_id(
```

Comment:
This could move to general utils perhaps

Reply:
I had it in `orchestrators/utils.py` before, but had to import it in many orchestrator files then. Would there be any explicit upside to having it in some utils file?

Reply:
I was thinking this could become a general util function `generate_uuid_from_str(str)` or similar

Reply:
but it's not that important

Reply:
Ah yeah that's a good idea, I'll still keep this method in the `BaseOrchestrator` but move the logic outside!

Reply:
Move it into `uuid_utils`